src/Hashids.net/Hashids.cs

```diff
@@ -138,14 +139,25 @@ public Hashids() : this(salt: string.Empty, minHashLength: 0, alphabet: DEFAULT_
         /// </summary>
         /// <param name="number">the number</param>
         /// <returns>the hashed string</returns>
-        public string EncodeLong(long number) => GenerateHashFrom(stackalloc[] { number });
+        public string EncodeLong(long number)
+        {
+            Span<char> result = stackalloc char[20];
```

Comment:
Why there is a limit of 20 chars? Now it throws if you try to encode value with `minHashLength` parameter set to >20, however previously it was possible to get string of any length

Reply:
I thought the max length for a long was 19 plus 1 for lottery hence 20.

We should have a test that validate any length.

Reply:
Also potentially any length does not make sense when it comes to max url length or other serialization issues or memory optimization.

Reply:
Don't confuse input number length (e.g `var num = 1234` has length of 4) and output hash length (which can differ)